src/Test/L1/Worker/L1TestBase.cs

```diff
@@ -108,21 +107,29 @@ protected static TaskStep CreateScriptTask(string script)
             var stringFile = Path.Combine(assemblyLocation, "en-US", "strings.json");
             StringUtil.LoadExternalLocalization(stringFile);
 
-            using (L1HostContext context = new L1HostContext("Agent", GetLogFile(this, testName)))
-            {
-                SetupMocks(context);
+            _l1hc = new L1HostContext("Agent", GetLogFile(this, testName));
+            SetupMocks(_l1hc);
 
-                // Use different working directories for each test
-                var config = GetMockedService<FakeConfigurationStore>();
-                config.WorkingDirectoryName = testName;
+            // Use different working directories for each test
+            var config = GetMockedService<FakeConfigurationStore>(); // TODO: Need to update this. can hack it for now.
+            config.WorkingDirectoryName = testName;
+        }
 
-                await SetupMessage(context, message);
+        protected L1HostContext _l1hc;
```

Comment:
nit: not a fan of abbreviated variable names. hostContext or l1hostcontext might be more readable.

Reply:
Renamed to `_l1HostContext`.